Marlin/src/core/utility.cpp

```diff
@@ -80,39 +80,39 @@ void safe_delay(millis_t ms) {
 
     #if HAS_BED_PROBE
       SERIAL_ECHOPAIR(
-        "Probe Offset X:" STRINGIFY(X_PROBE_OFFSET_FROM_EXTRUDER)
-                    " Y:" STRINGIFY(Y_PROBE_OFFSET_FROM_EXTRUDER)
-                    " Z:", zprobe_zoffset
+        "Probe Offset X:" STRINGIFY(zprobe_offset[X_AXIS])
```

Comment:
We cannot `STRINGIFY` code in C++. Only preprocessor defines.

Reply:
The replace left () that was around the defined value, where it was also just a single variable. I left it assuming it was desired formatting since it was consistent. As for the stringify. It didn't present a functional issue with the output and I didn't dig into what the macro did. Output works regardless.

Reply:
`STRINGIFY` is a fairly common Macro in C++-land. It gives you the literal result of a `#define` evaluation as a quoted string.